src/ucs/config/parser.c

```diff
@@ -1625,13 +1618,45 @@ void ucs_config_parser_warn_unused_env_vars(const char *prefix)
         }
         ucs_warn("unused env variable%s:%s%s (set %s%s=n to suppress this warning)",
                  (num_unused_vars > 1) ? "s" : "", unused_env_vars_names,
-                 truncated ? "..." : "", UCS_CONFIG_PREFIX,
+                 truncated ? "..." : "", UCS_DEFAULT_ENV_PREFIX,
```

Comment:
print right prefix here?

Reply:
UCX_ prefix is used to control this behavior regardless of env_prefix